docs/contributing.md

```diff
@@ -0,0 +1,69 @@
+
+If not, you'll need to create a new Issue. Help us to help fix the Issue quicker by following these guidelines. Your Issue should contain:
+
+#### A description of the Issue in object-deviation format
+
+Object-deviation format is a very specific way of describing faults. The more specific you can be, the easier it is for us to fix the fault. Begin with the _object_, such as function, a skill, or a hardware feature, then explain the _deviation from the expected condition_.
+
+For example:
+
+* "When my Mycroft Mark 1 device \(**object**\) powers on, the color of his eyes is red instead of blue \(**deviation - red - from expected condition - blue**\)
+* "When I ask about the weather \(**object**\), Mycroft reports the low temperature correctly, but the high temperature is incorrectly reported as the low temperature \(**deviation - incorrect temperature - from expected condition - correct temperature**
+
+### Making changes
+
+1. [Fork the Project](https://help.github.com/articles/fork-a-repo/)
+2. [Create a new Issue](https://help.github.com/articles/creating-an-issue/) if one doesn't already exist.
+3. Create a **feature** or **bugfix** branch based on **dev** with your issue identifier. For example, if your issue identifier is: **issue-123** then you will create either: **feature/issue-123** or **bugfix/issue-123**. Use **feature** prefix for issues related to new functionalities or enhancements and **bugfix** in case of bugs found on the **dev** branch
+4. Make sure you stick to the coding style and OO patterns that are used already.
+5. Document code using [Google-style docstrings](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).  All functions and class methods that are expected to be called externally should include a docstring.  \(And those that aren't [should be prefixed with a single underscore](https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references)\).
+6. Make commits in logical units and describe them properly. Use your issue identifier at the very begin of each commit. For instance:
+
+   `git commit -m "Issues-123 - Fixing 'A' sound on Spelling Skill"`
+
+7. Once you have committed everything and are done with your branch, you have to rebase your code with **dev**. Do the following steps: 1. Make sure you do not have any changes left on your branch 2. Checkout on dev branch and make sure it is up-to-date 3. Checkout your branch and rebase it with dev 4. Resolve any conflicts you have 5. You will have to force your push since the historical base has changed 6. Suggested steps are:
+
+   ```text
+      git checkout dev
+      git fetch
+      git reset --hard origin/dev
+      git checkout <your_branch_name>
+      git rebase dev
+      git push -f
+   ```
+
+8. Once everything is OK, you can finally [create a Pull Request \(PR\) on Github](https://help.github.com/articles/using-pull-requests/) in order to be reviewed and merged.
```

Comment:
...create a Pull Request on Github. Most repositories have automated tests that will run at this time; if they all pass, your PR is ready to be reviewed and merged, pending any review feedback.